### DISCUSSION TITLE
Can I run a macOS guest newer than my host?

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#5715] [UTM Help (macOS)] [answered]
Hi community,

I use UTM on my MacBook with macOS 12.6. I would like to install a newer version of macOS on UTM, like the macOS 14 beta.
I don't know if this is possible, and I don't want to waist my time downloading a 12-14GB .ipsw file for nothing!

If needed, my MacBook can be upgraded to a newer version although I am not permitted to do so yet from my admin.

Thanks

**Comment by mounta11n** [comment, marked as answer]
Yes of course this is absolutely possible. There are no restrictions between software <-> software. The only you have to consider is your hardware. Technically you could have hardware <-> software ‚restrictions‘ due to lack of performance. „Restrictions“ means here in most cases that you still could get the vm run but maybe very slow and not usable for everday tasks.
